clients/security_config.go

```diff
@@ -73,3 +78,26 @@ func (c *SecurityConfig) Build() (*tls.Config, error) {
 
 	return configuration, nil
 }
+
+// Client creates an http.Client from the SecurityConfig instance.
+func (c *SecurityConfig) Client() (*http.Client, error) {
```

Comment:
Why add functionality to a deprecated API?

Reply:
Because ooops.